### DISCUSSION TITLE
Best way to skip tests that rely on grids

### DISCUSSION THREAD

**Discussion by nezos** [pyproj4/pyproj#858] [Q&A] [answered]
I am making an effort to introduce pyproj in Void linux. In void the grid files have not been packaged and hence are not available. 

Is there something I could modify in the source files to skip all tests related to grid files rather than adding code to skip each test seperately?

Thanks

**Comment by snowman2** [comment, marked as answer]
You can use the markers: https://github.com/pyproj4/pyproj/blob/master/pytest.ini

```
pytest -m "not grid"
```
